public/platform.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="../css/platform.css">
+  <script type="text/javascript" src="../js/platform.js">
+
+  </script>
+  <title>PlatForm Game</title>
+</head>
+
+<body>
+
+  <script>
+  runGame(GAME_LEVELS, DOMDisplay);
```

Comment:
I don't think you need to do this since you're already requiring your javascript file on line 7.